src/Sylius/Behat/Context/Api/Shop/AddressContext.php

```diff
@@ -151,6 +158,14 @@ public function iDeleteTheAddress(string $fullName): void
         $this->addressClient->delete($id);
     }
 
+    /**
+     * @When /^I try to delete (address belongs to "([^"]+)")$/
+     */
+    public function iDeleteTheAddressBelongsTo(AddressInterface $address): void
+    {
+        $this->addressClient->delete($this->iriConverter->getIriFromItem($address));
```

Comment:
Shouldn't it looks like:
```suggestion
        $this->addressClient->delete($address->getId());
```